src/commands/synthetics/reporters/default.ts

```diff
@@ -275,6 +275,10 @@ const getAttemptSuffix = (passed: boolean, retries: number, test: Test) => {
     return ''
   }
 
+  if (timedOutRetry) {
+    return ` (attempt ${currentAttempt + 1} of ${maxAttempts})`
```

Comment:
I'm not sure to understand this equation: `currentAttempt + 1`, given that `currentAttempt` is already `retries + 1`.
There could be several fast retries timing out, no? So it wouldn't be `+ 1` but `+ n`, no?

Reply:
We don't need `+ n` because we are talking about the retry that was expected directly after the current attempt, not _all_ of the future retries.